### ISSUE TITLE
[Request] Print Speed Adjust on the Fly

### ISSUE THREAD

**Issue by dcwalmsley** [foosel/OctoPrint#1663]
After uploading and slicing STL file, I find myself needing to adjust the print speed while printing.  Looking for a slider function added under Control tab that allows adjust speed in percentage with 100 percent being the default speed allowing plus or minus 50% (ie, 50-150%).

**Comment by BerndJM** [comment]
There is already a slider "Feedrate" on the control tab - isn't it that what you're looking for?

**Comment by dcwalmsley** [comment]
No.  Looking for a means to increase or decrease mm/s printer speed.  
Instead of terminating a partial print and reslicing at a different speed 
I'm looking to make print speed adjustments on the fly.

**Comment by BerndJM** [comment]
> Print speed adjustments on the fly
Call me stupid, but thats exactly what this slider does ...

**Comment by dcwalmsley** [comment]
Oh...  OK.  I thought it had a different function.  Maybe reword it to 
Print Speed vice Feed Speed.

I apologize for my ignorance.

**Comment by foosel** [comment]
As already pointed out, that's already there. Renaming it would make it inaccurate - it is a feed rate modifier.
